concepts/numbers/introduction.md

```diff
@@ -41,4 +41,35 @@ When one of the involved types is a floating-point type, the result will also be
 `5.0 / 2` and `5 / 2.0` will return `2.5`.
 `%` is the remainder operator and will return the remainder of an integer division: `5%3` is `2`.
 
+## Assignment operator
+
+The assignment operator assigns a variable with a literal.
+Always takes place from right to left, and never the other way around.
+
+```cpp
+int x, y;
```

Comment:
Please avoid uninitialized variables in Examples.
-> https://github.com/exercism/cpp/blob/main/concepts/basics/introduction.md